app/src/main/java/de/xikolo/config/Feature.kt

```diff
@@ -34,4 +34,8 @@ object Feature {
         }
     }
 
+    @TargetApi(25)
+    fun shortcuts(): Boolean {
+        return Build.VERSION.SDK_INT >= 25
+    }
```

Comment:
Is this the correct way to implement this as a function? @tobiasrohloff gave the feedback, that it should be in a function.

Reply:
Then you misunderstood me. Make this a `const val`. We only provide flags here. Why do you added the `TargetApi` annotation? This flag needs to be called with all API verions.

Reply:
Jup, misunderstood that comment: https://github.com/openHPI/xikolo-android/pull/227#discussion_r408111190